src/libsync/syncresult.cpp

```diff
@@ -51,19 +51,19 @@ QString SyncResult::statusString() const
         re = QLatin1String("Error");
         break;
     case SetupError:
-        re = QLatin1String("SetupError");
+        re = QLatin1String("Setup error");
         break;
     case SyncPrepare:
-        re = QLatin1String("SyncPrepare");
+        re = QLatin1String("Sync prepare");
         break;
     case Problem:
         re = QLatin1String("Success, some files were ignored.");
         break;
     case SyncAbortRequested:
-        re = QLatin1String("Sync Request aborted by user");
+        re = QLatin1String("Sync request aborted");
```

Comment:
"Sync request canceled" could work here too

Reply:
Yup, changed it to cancelled here too